pkg/apiserver/sync/convert.go

```diff
@@ -49,7 +50,8 @@ func (c *CR2UX) ConvertApp2DatastoreApp(ctx context.Context, targetApp *v1beta1.
 			model.LabelSourceOfTruth:  model.FromCR,
 		},
 	}
-
+	appMeta.CreateTime = targetApp.CreationTimestamp.Time
+	appMeta.UpdateTime = time.Now()
```

Comment:
Should we check whether `targetApp` was updated?

Reply:
if the app is not updated, can not execute here code.